packages/app/src/cli/services/dev/graphiql/server.ts

```diff
@@ -149,6 +151,7 @@ export function setupGraphiQLServer({
         {
           url: localhostUrl,
           defaultQueries: [{query: defaultQuery}],
+          query,
```

Comment:
What about instead:

```suggestion
          defaultQueries: [{query: query ?? defaultQuery}],
```

And then you can skip the template changes below.

Reply:
Hmmmm, actually I do see that the behavior is different, where if we open with a query it gets added to the currently open tabs, vs. replacing the defaults. So, I think what you did makes sense.

Reply:
I originally did exactly what you said :D but yeah in the end they support this separate `query` attribute for a reason.